core/config/config.go

```diff
@@ -49,15 +49,15 @@ var (
 	// This is the default, Chain development configuration.
 	// These options can be updated with build tags.
 	BuildConfig = struct {
-		LoopbackAuth bool `json:"is_loopback_auth"`
-		MockHSM      bool `json:"is_mockhsm"`
-		Reset        bool `json:"is_reset"`
-		PlainHTTP    bool `json:"is_plain_http"`
+		LocalhostAuth bool `json:"is_localhost_auth"`
```

Comment:
Are you sure? There are only three spots where this variable is mentioned. I think it will be less confusing if we can make the language as consistent as we can. For example, here are two of the references:

<img width="689" alt="screen shot 2017-05-01 at 2 06 21 pm" src="https://cloud.githubusercontent.com/assets/464857/25594892/6daf32a4-2e77-11e7-958b-c0676c204b24.png">

<img width="458" alt="screen shot 2017-05-01 at 2 06 11 pm" src="https://cloud.githubusercontent.com/assets/464857/25594895/70aa12b2-2e77-11e7-8de4-462ac72034a6.png">

If I keep the existing variable name `LoopbackAuth`, then someone who is unfamiliar with this may think the discrepancy is a mistake.